DSCResources/VE_XD7Administrator/VE_XD7Administrator.psm1

```diff
@@ -194,14 +195,13 @@ function Set-TargetResource {
 
         if ($Credential) {
             AddInvokeScriptBlockCredentials -Hashtable $invokeCommandParams -Credential $Credential;
+            Write-Verbose ($localizedData.InvokingScriptBlockWithParams -f [System.String]::Join("','", @($Name, $Enabled, $Ensure)));
```

Comment:
Can you move the Write-Verbose call above the `if ($Credential) {` call and **the same for all other instances** across all these files?